.github/workflows/tests.yaml

```diff
@@ -258,7 +258,7 @@ jobs:
             path: tests/cashiers-check
           - cmd: cd tests/typescript && anchor test
             path: tests/typescript
-          - cmd: cd tests/zero-copy && anchor test
+          - cmd: cd tests/zero-copy && anchor test && cd programs/zero-copy && cargo test-bpf
```

Comment:
instead of adding it here, could try adding this in the `Anchor.toml` file of the `zero-copy` test repo under the `[scripts]` tab

Reply:
tried it, didnt work